common/predictive-text/worker/correction/distance-modeler.ts

```diff
@@ -237,6 +274,8 @@ namespace correction {
   // The set of search spaces corresponding to the same 'context' for search.
   // Whenever a wordbreak boundary is crossed, a new instance should be made.
   export class SearchSpace {
+    static readonly EDIT_DISTANCE_COST_SCALE = 5;
```

Comment:
I feel like the comments in `currentCost()` could be helpful here instead (or in addition!) — why was 5 chosen? Well, that's answered in `(SearchNode || SearchEdge).currentCost()`!

Reply:
Went ahead and edited that into #3565.